### PR TITLE
Add changelog for 0.11

### PR DESCRIPTION
**Pull request by HeroicKatora**

### CODE REVIEW

**Comment by fintelia** [comment]
I think the README example might still need to be updated for the new Decoder interface?

**Comment by HeroicKatora** [comment]
Good point, took the opportunity to include the readme as a doctest with the same hack as in `image`.

File: README.md [hunk 1]
```diff
@@ -10,20 +10,18 @@ This library provides all functions necessary to de- and encode GIF files.
...
-use gif::SetParameter;
-let mut decoder = gif::Decoder::new(File::open("tests/samples/sample_1.gif").unwrap());
+let input = File::open("tests/samples/sample_1.gif").unwrap();
 // Configure the decoder such that it will expand the image to RGBA.
-decoder.set(gif::ColorOutput::RGBA);
+let mut options = gif::DecodeOptions::new();
+options.set_color_output(gif::ColorOutput::RGBA);
 // Read the file header
-let mut decoder = decoder.read_info().unwrap();
+let mut decoder = options.read_info(input).unwrap();
```
Comment (fintelia) [inline README.md:24]: `read_info` isn't the most obvious method naming here, but I don't think it is a big deal

**Review by fintelia — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HeroicKatora) [reply README.md:24]: Maybe. It was the previous name, we can always add an alias for `open`. Not a big deal in terms of interface.

**Pull request merged**